README.md

```diff
@@ -79,7 +82,7 @@ source /opt/ros/dashing/setup.bash
 source ~/ros2_ws/install/local_setup.bash
 
 cd ~/ros2_ws
-ros2 run realsense_node realsense_camera_node __params:=src/ros2_intel_realsense/realsense_ros/config/d435.yaml __ns:=/d435
+ros2 run realsense_node realsense_camera_node __params:=./src/ros2_intel_realsense/realsense_ros/config/d435.yaml __ns:=/d435
```

Comment:
why using realsense_camera_node instead of realsense_node? I suggest using the consistent name.

Reply:
__params:=./src/ros2_intel_realsense/realsense_ros/config/d435.yaml
suggest you use this command as below:
__params:=\`ros2 pkg prefix realsense_examples\`/share/realsense_ros/config/d435.yaml